src/modals/CenteredModal.tsx

```diff
@@ -60,46 +62,44 @@ const CenteredModal: FC<CenteredModalProps> = ({
   return (
     <ModalContainer onClose={onClose} focusMode={focusMode} hasPadding>
       <CenteredBox role="dialog" {...fadeInOutScaleFast} narrow={narrow}>
-        <ModalHeader contrast={!!header}>
-          <TitleRow>
-            {onBack && (
-              <BackButton aria-label={t('Back')} squared role="secondary" transparent onClick={onBack} borderless>
-                <ChevronLeft />
-              </BackButton>
-            )}
-            <PanelTitle size="small" useLayoutId={false}>
-              <span ref={elRef} tabIndex={0} role="heading">
-                {title}
-              </span>
-              {subtitle && <ModalSubtitle>{subtitle}</ModalSubtitle>}
-            </PanelTitle>
-            <CloseButton aria-label={t`Close`} squared role="secondary" transparent onClick={onClose} borderless>
-              <X />
-            </CloseButton>
-          </TitleRow>
-          {header && <ModalHeaderContent>{header}</ModalHeaderContent>}
-        </ModalHeader>
-        {dynamicContent ? children : <ScrollableModalContent>{children}</ScrollableModalContent>}
-
-        {isLoading && (
-          <>
-            <ModalBackdrop light />
-            <ModalLoadingSpinner>
-              <Spinner />
-            </ModalLoadingSpinner>
-          </>
-        )}
+        <Scrollbar translateContentSizeYToHolder isDynamic>
```

Comment:
Crap. I don't remember exactly, I had many problems with this Scrollbar 🙃 I intend to come back to this send modal anyway, I'll take a second look during the weekend.

Reply:
I don't like the library we are using, I also faced many issues. It's also not well maintained. But I can't find any good alternatives.

Reply:
Crazy that no other good libraries seem to exist. I'll think about this.

Reply:
I'm a bit hesitant to merge this PR with this change cause I remember it took me a long time to figure out the solution to the dynamic height of the modal :/ Are you sure you wanna merge this as it is? Is the problem that this change solves harder to fix than the dynamic scroll content? (I know you cannot answer that question, but that's what it's in my mind)

Reply:
Changes reverted in https://github.com/alephium/desktop-wallet/pull/614 in case you are interested. It fixes the accessibility problems with all popups. The only drawback I noticed is that the modal header is always visible when scrolling. I would prefer to include a fix for that if needed in a next PR instead of fixing the popups in a next PR, WDYT?